### PR TITLE
ZERO138 update module name, add summary to makefile

### PR DESCRIPTION
**Pull request by cjhawkins**

### CODE REVIEW

File: zero-module.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-name: aws-eks-stack
```
Comment (davidcheung) [inline zero-module.yml:1]: 👍  didn't realize the other modules depends on `zero-aws-eks-stack`, was wondering why they weren't running haha

File: Makefile [hunk 2]
```diff
@@ -2,3 +2,10 @@
 run:
 	cd $(PROJECT_DIR) && AUTO_APPROVE="-auto-approve" make
 
+summary:
+	@echo "zero-aws-eks-stack:"
+	@echo "- Repository URL: ${REPOSITORY}"
+	@echo "- To see your kubernetes clusters, run: 'kubectl config get-contexts'"
+	@echo "- To switch to a cluster, use the NAME from the previous command in 'kubectl config use-context NAME'"
```
Comment (bmonkman) [inline Makefile:9]: Actually why don't we just show them the correct cluster name here?

Comment (cjhawkins) [reply Makefile:9]: You want to see separate `kubectl config use-context`'s for each environment that is applied?

Comment (bmonkman) [reply Makefile:9]: Yeah

**Pull request merged**